events/models.py

```diff
@@ -29,6 +29,7 @@ class Event(models.Model):
         default=True,
         help_text="If this is checked all users that attends the event needs to \
         be accepted by an admin.")
+    external_signup_url = models.CharField(max_length=80, blank=True)
```

Comment:
I haven't looked into this so there is probably some valid case for using a CharField, but why not use a URLField instead? Seem to do the same thing from the docs? https://docs.djangoproject.com/ja/1.10/ref/models/fields/#urlfield Please enlighten me <3

Also, 80 chars for a URL is not a problem per se, but the hassle it would bring if you needed to enter a longer URL would not be worth the few bits and bytes we save. But perhaps there's some other explanation for this? :)